packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -94,6 +94,7 @@ export class ObjectForest implements IEditableForest {
 
 	public constructor(
 		public readonly anchors: AnchorSet = new AnchorSet(),
+		private readonly additionalAsserts: boolean = false,
```

Comment:
I don't see any harm from making this public since this class is internal, and we have a separate interface type for when we don't want to expose details specific to this forest type. That will make ensuring this is set or not easier, avoiding needs for unsafe casts like in your test.

Reply:
Updated `additionalAsserts` field to public, and removed casts from the tests